content/id/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

```diff
@@ -0,0 +1,449 @@
+
+Banyak _pipeline_ metrik memungkinkan kamu mendeskripsikan metrik baik berdasarkan nama atau kumpulan tambahan
+deskriptor yang disebut _labels_. Untuk semua jenis metrik non-sumber daya (pod, objek, dan eksternal,
+dijelaskan seperti dibawah), kamu dapat menentukan _selector_ label tambahan yang diteruskan ke pipa metrik kamu. Misalnya, jika kamu mengumpulkan metrik `http_requests` dengan label `verb`
+, kamu dapat menentukan blok metrik berikut untuk diskalakan hanya pada permintaan GET:
+
+```yaml
+type: Object
+object:
+  metric:
+    name: http_requests
+    selector: {matchLabels: {verb: GET}}
+```
+
+_Selector_ ini menggunakan sintaks yang sama dengan _selector_ lengkap label Kubernetes. Pipa pemantauan
+menentukan cara mengecilkan beberapa seri menjadi satu nilai, jika nama dan pemilih cocok dengan 
+beberapa seri. Selektor bersifat aditif, dan tidak dapat memilih metrik yang mendeskripsikan objek 
+yang **bukan** objek target (target pod dalam kasus tipe `Pod`, dan objek yang dijelaskan dalam kasus tipe `Objek`).
+
+### Penyekalaan otomatis pada metrik yang tidak terkait dengan objek Kubernetes
+
+Aplikasi yang berjalan di Kubernetes mungkin perlu melakukan penyekalaan otomatis berdasarkan metrik yang tidak
+memiliki hubungan yang jelas dengan objek apa pun di klaster Kubernetes, seperti metrik yang mendeskripsikan
+layanan yang dihosting tanpa korelasi langsung dengan namespace Kubernetes. Di Kubernetes 1.10 dan yang lebih baru, kamu dapat menangani kasus penggunaan ini dengan *metrik eksternal*.
+
+Menggunakan metrik eksternal membutuhkan pengetahuan tentang sistem pemantauanmu; penyiapannya mirip dengan yang diperlukan saat menggunakan metrik khusus. Metrik eksternal memungkinkan kamu menskalakan klaster kamu secara otomatis berdasarkan metrik apa pun yang tersedia di sistem pemantauanmu. Cukup berikan blok `metrik` dengan `nama` dan `pemilih`, seperti di atas, dan gunakan jenis metrik `Eksternal`, bukan `Objek`.
+Jika beberapa _series_ cocok dengan `metricSelector`, jumlah dari nilai mereka akan digunakan oleh HorizontalPodAutoscaler.
+Metrik eksternal mendukung jenis target `Value` dan `AverageValue`, yang berfungsi persis sama seperti saat kamu menggunakan tipe `Object`.
+
+Misalnya, jika aplikasi kamu memproses tugas dari layanan antrian yang dihosting, kamu dapat menambahkan bagian berikut ke manifes HorizontalPodAutoscaler untuk menentukan bahwa kamu memerlukan satu pekerja per 30 tugas yang belum diselesaikan.
+
+```yaml
+- type: External
+  external:
+    metric:
+      name: queue_messages_ready
+      selector: "queue=worker_tasks"
+    target:
+      type: AverageValue
+      averageValue: 30
+```
+
+Jika memungkinkan, lebih baik menggunakan target metrik khusus daripada metrik eksternal, karena lebih mudah bagi administrator klaster untuk mengamankan API metrik khusus. API metrik eksternal berpotensi memungkinkan akses ke metrik apa pun, jadi administrator kluster harus berhati-hati saat mengeksposnya.
```

Comment:
typo?
```suggestion
Jika memungkinkan, lebih baik menggunakan target metrik khusus daripada metrik eksternal, karena lebih mudah bagi administrator klaster untuk mengamankan API metrik khusus. API metrik eksternal berpotensi memungkinkan akses ke metrik apa pun, jadi administrator klaster harus berhati-hati saat mengeksposnya.
```

Reply:
Sorry, I was mean to translate it into klaster